driver-dse/src/main/java/com/datastax/driver/graph/AbstractGraphStatement.java

```diff
@@ -0,0 +1,141 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.graph;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+
+import com.datastax.driver.core.Statement;
+import com.datastax.driver.core.exceptions.InvalidQueryException;
+
+abstract class AbstractGraphStatement<T extends Statement> {
+
+    protected final Map<String, ByteBuffer> payload;
+    protected T wrappedStatement;
+    protected final GraphSession session;
```

Comment:
This was part of the changes to adapt for the 3.0 way of creating simple statements. I am correcting this today.